utils/build-script

```diff
@@ -961,20 +961,16 @@ class BuildScriptInvocation(object):
 
         # Non-build-script-impl products...
         # Note: currently only supports building for the host.
-        for host_target in [self.args.host_target]:
+        for host_target in [
```

Comment:
I think you are asking why have a `for ... in` for a known one element list, right? I think the reason is future-proofing, because the comment says that the products only support building for the host, but they might some day support building for more targets, and that change will be easy having this in place. Ben might know the real reason, since he wrote the original code.